package.json

```diff
@@ -45,6 +46,7 @@
     "flow-bin": "0.57.3",
     "glob": "^7.1.2",
     "jest": "^22.4.2",
+    "lerna": "^2.11.0",
```

Comment:
What do you think about adding `yarn` as a dev-dependency?  This gives us two things.  First, developers wont have to brew install yarn just to work on wonder-blocks.  Second, we can control what version of yarn is used, so if there is an issue in a particular version we wont have to worry about inconsistencies between dev environments.  We can add that as a separate PR too if you want to give it a try to make sure things work as expected.

Reply:
I was planning to create a separate PR to install `yarn` as a dep.

Reply:
We should consolidate our code and docs to specify `yarn` instead of `npm` too. No reason to have both.